app/Http/Controllers/Dashboard/SettingsController.php

```diff
@@ -384,6 +385,10 @@ public function postSettings()
             Lang::setLocale(Binput::get('app_locale'));
         }
 
+        if (Binput::has('always_authenticate')) {
+            Artisan::call('route:cache');
```

Comment:
A workaround I've had to implement because the `cachet:setup` command caches the routes. After enabling the `always_authenticate` feature your routing cache becomes invalid. If the original approach on solving this issue is correct. What should we do with this code? Some idea's: 

1. Use DI for resolving Artisan or even the `route:cache` command?
2. Let caches be handled by a different `service` that listens to some `settingsupdated` event. Because this controller has too many responsibilities.